docs/guide/field_type_reference.md

```diff
@@ -1,12 +1,15 @@
 # Field Types reference
 
+A Field Type is the smallest entity of storage in eZ Platform. It determines how a specific type of information is validated, stored, retrieved, formatted and so on.
```

Comment:
I know this is form text that was here before, but this is not entirely correct for a few reasons:
- Field Type consists of two entities: Field Value and Field Definition, I guess you where referring to Field Value
- super nitpic: It's not the smallest, a hard coded integer column would be smaller


So I guess we should reword, what would we like to explain here as an abstract? That it's the underlying building block of content model/modeling or something like that? Text below kind of says that so guess we can make it even shorter.